packages/clarity-js/src/data/metadata.ts

```diff
@@ -28,6 +32,12 @@ export function start(): void {
   metric.max(Metric.ClientTimestamp, ts);
   metric.max(Metric.Playback, config.lean ? BooleanFlag.False : BooleanFlag.True);
 
+  // Read cookies specified in configuration
```

Comment:
What is this for? What cookies would be configured and why?

Reply:
Will forward an offline thread on this.